test/metabase/api/database_test.clj

```diff
@@ -299,7 +299,7 @@
                                                              :visibility_type   "normal"
                                                              :has_field_values  "list"
                                                              :database_position 1
-                                                             :database_required false})]
+                                                             :database_required true})]
```

Comment:
Now that `test-data.categories.name` is `NOT NULL` I had to update two metadata tests.